### DISCUSSION TITLE
dags not showing up in Dags list

### DISCUSSION THREAD

**Discussion by mashirali** [apache/airflow#21312] [General]
### Apache Airflow version

2.2.3 (latest released)

### What happened

I compiled the source code and then added dag files in dags folder.. but when in ran the breeze env. it didn;t shown in list i-e list is showing empty.

### What you expected to happen

Dags should shown up in DAGs list

### How to reproduce

compile the source code in pycharm

### Operating System

windows 11

### Versions of Apache Airflow Providers

2.2.3

### Deployment

Virtualenv installation

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
If you want some help, please describe your setup. You seem to mix Breeze and virtualenv. You don't specify what do you mean by dags folder. I propose you read https://github.com/apache/airflow/blob/main/BREEZE.rst and follow it.

**Reply by mashirali** [reply]
I have put my .py files in dag folder which are located in /root/dags. As per the way things work in airflow, it has to show dags in UI table on home page. But Nothing is there
![dag](https://user-images.githubusercontent.com/57099739/152483112-4b19a6de-9a35-43f8-ac64-315be599f852.png)

**Reply by potiuk** [reply]
That's not nearly enough,.

**Reply by mashirali** [reply]
i have dowloaded official docker compose file and tried to build image and after i make the docker compose up.. everything works fine but the changes i made in the code are not there its just the same working airflow as we can download and install from package maanger. wht about my custom changes on source code how can i make them work?

**Reply by potiuk** [reply]
- How did you apply your changes? 
- You know it's docker compose so airflow runs in docker? 
- Where did you and how did you modify your code?
- Have you followed https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html and have you seen where to put your dags?
- How does your dag code look like?
- What errors do you see in logs when you try to add the code?
- what kind of source code modification you did ?
- where did you do them?
- how did you make sure your changes were added ? 
- how did yout build the image?
- how did you add your custom code to the image?

**Reply by potiuk** [reply]
We cannot read your mind and understand what you have done until you tell us what you did. This is not even close to even remotely trying to understand what you were trying to do.

**Comment by teu** [comment]
We have the same problem with this airflow version.

The dags are available when URL is entered directly, however they are not being scheduled not are visible in dag list. I have run the setup locally (same config) and see.

**Comment by mashirali** [comment]
Can you explain your problem with screen shots

________________________________
